hack/testing-olm/test-657-im-block-autocreate-for-write-suffix.sh

```diff
@@ -51,7 +51,7 @@ if [ "${DO_SETUP:-true}" == "true" ] ; then
   expect_success "oc -n ${TEST_NAMESPACE} create -f ${repo_dir}/hack/cr.yaml"
 
   log::info "Waiting for elasticsearch-operator to deploy the cluster..."
-  try_until_success "oc -n ${TEST_NAMESPACE} get deployment -l component=elasticsearch -o jsonpath={.items[0].metadata.name}) --ignore-not-found" $((2 * $minute))
+  try_until_success "oc -n ${TEST_NAMESPACE} get deployment -l component=elasticsearch -o jsonpath={.items[0].metadata.name})" $((2 * $minute))
```

Comment:
@jcantrill seems like this "ignore-not-found" flag was wrong, you really want this to wait until the deployment exists.  it was causing at some least flake failures.

But it also looks like you attempted to remove this test entirely recently, but you only removed it from hack/testing, not hack/testing-olm (the new dir i created to run the tests via a proper olm deployment path).

So if you want this test disabled entirely, let's discuss doing that tomorrow.

Reply:
Per our conversation your observation is correct and we should not have the "ignore-not-found" flag